src/connection/database/index.js

```diff
@@ -90,7 +90,7 @@ function processDatabaseOptions(opts) {
     additionalSignUpFields = undefined;
   } else if (additionalSignUpFields) {
     additionalSignUpFields = additionalSignUpFields.reduce((r, x) => {
-      let { icon, name, options, placeholder, prefill, type, validator, value } = x;
+      let { icon, name, options, placeholder, prefill, type, validator, value, isExtra } = x;
       let filter = true;
 
       const reservedNames = ['email', 'username', 'password'];
```

Comment:
should we update reservedNames to include extraFields?

Reply:
Done